### ISSUE TITLE
Show origin of calls for analytics

### ISSUE THREAD

**Issue by philippeluickx** [apinf/api-umbrella-dashboard#1072]
The (main part of the) url that does the call to the API endpoint. Let's say abc.com uses the API of xyz.com, then we should show this info to the xyz.com owner. Include the count for that time period as well.

**Comment by brylie** [comment]
Please give a specific example using Apinf.io.

**Comment by philippeluickx** [comment]
This is similar to google analytics. You want to know where users come from. So let's say "apicatalog.com" is a webapp that uses the apinf.io API to show a list of APIs, then I as a API owner would like to see that this webapp exists and how often they use my API.

The value is that you want to discover which use cases people implement for your API. If you only have a username (the person that registered the key), then you probably still know nothing. Even worse, if you can use an API without key, you know nothing at all.

**Comment by brylie** [comment]
API calls come directly from the application IP address, and may not contain information such as [HTTP 'referer'](https://en.wikipedia.org/wiki/HTTP_referer). How might our proxy get the referring application name, domain name, or other relevant metadata?

**Comment by philippeluickx** [comment]
Good point, to be investigated? Also, this might be data that is saved on
the proxy level?

**Comment by brylie** [comment]
Yep, API Umbrella saves at least the following related data:
- IP address
  - geolocation
- API Key
- user

![API Umbrella screenshot](https://github.com/apinf/api-umbrella-dashboard/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+assignee%3Abrylie+milestone%3A%22Sprint+23%22)

Our analytics data comes straight from the paroxy, so we are somewhat limited to what it collects.
